.github/workflows/download_translations.yml

```diff
@@ -23,16 +23,29 @@ jobs:
           upload_sources: false
           upload_translations: false
           download_translations: true
-          localization_branch_name: l10n_crowdin_translations
-
-          create_pull_request: true
-          pull_request_title: 'chore: Update translations'
-          pull_request_body: 'New Crowdin pull request with translations'
-          pull_request_labels: 'T-chore, A-i18n'
-          pull_request_base_branch_name: 'master'
-          commit_message: 'chore: Update translations from Crowdin'
+          push_translations: false
+          create_pull_request: false
         env:
           CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+
+      - name: Commit & push
+        run: |
+          git config user.name "RuffleBuild"
+          git config user.email "ruffle@ruffle.rs"
+          git checkout -b l10n_crowdin_translations
+          git add -A
+          git commit -m 'chore: Update translations from Crowdin'
+          git push --set-upstream origin l10n_crowdin_translations --force
```

Comment:
Won't this command also require some kind of authentication/authorization; like the next via `$GITHUB_TOKEN`? :thinking:

Reply:
It's possible I messed something up, I'll replace that with the `ad-m/github-push-action` action that is used in other places and test

Reply:
done